pg_search/src/bootstrap/create_bm25.rs

```diff
@@ -29,11 +29,12 @@ use super::format::format_hybrid_function;
 // but we need to account for the trailing _bm25_index suffix
 const MAX_INDEX_NAME_LENGTH: usize = 52;
 
-#[pg_extern(sql = "
+#[pg_extern(
+    sql = "
 CREATE OR REPLACE PROCEDURE paradedb.create_bm25(
-    index_name text DEFAULT '',
-    table_name text DEFAULT '',
-    key_field text DEFAULT '',
+    index_name text,
+    table_name text,
+    key_field text,
```

Comment:
We have these `DEFAULT` calls here so that we can validate the arguments ourselves and give the user a more helpful error message. If you remove them, people will just see a confusing "create_bm25 does not exist" when they forget a `key_field`.

Reply:
Now that we overload `create_bm25` to accept both JSON and TEXT `_fields`, if the user doesn't pass in one of the required options they'll get a "ambiguous function" error. I'm open to ideas here...

Reply:
Hmm I see, that's a pretty high price to pay. `create_bm25` is our most complex function, and the first one a brand new user needs to use. I believe we should be adding validation, not removing it. Let's brainstorm a workaround over the weekend.